### ISSUE TITLE
Website Updates

### ISSUE THREAD

**Issue by windauer** [GF-Corporate-Archives/gf-johann-conrad-fischer#420]
In diesem Tickets werden in den Kommentaren Aktualisierungen der Webseite vermerkt.

**Comment by windauer** [comment]
Current live version is: eb5d8940166f05a85e72b06ef1d2136618dd8d67

**Comment by windauer** [comment]
Current live version is: ee3f6c43ef7a7af13dc6fb35271b4fb19e52d8b8

**Comment by windauer** [comment]
Current live version is: c06ab538a836d2e41aa10934c7ba668760fda7ce

**Comment by windauer** [comment]
Current live version is: 7cfa5c4a7afb42228c9a36420e9e47f37c2c68fa

**Comment by windauer** [comment]
Current live version is: c6c4003a5d38d5d8822e345cac4869af3f4e0cf4

**Comment by windauer** [comment]
Current live version is: cf9031c3c637c88fd74928276cd59d4953f097a8

**Comment by windauer** [comment]
Current live version is: a793b77429aca22011afd8e67b8e694725f9875d

**Comment by windauer** [comment]
Current live version is: 82e9a7961d6310d3a332a662c33cfd8bd9bdae70 (highlights)

**Comment by windauer** [comment]
Current live version is: 57020fd0ef78fa8742c3c52fb4f0f0496314f49d (Journey title page)

**Comment by windauer** [comment]
Current live version is: 55ff0147972da7416d809db3025141bfabc79cb4  (includes data update till 2023-06-05 12:52)

**Comment by feggimann** [comment]
@windauer Ja, das ist für uns ok. Dann schauen wir ab jetzt nur noch https://johannconradfischer.sources-online.org an.

**Comment by windauer** [comment]
Current live version at https://johannconradfischer.sources-online.org is:  ba451aca227ad885813385ec3eb24dc98b4d335e

**Comment by windauer** [comment]
Current live version at https://johannconradfischer.sources-online.org/ is: 
10c5a4f900555738e1efcc1abf70d548e82aed98 (includes latest data from this repo and Anton)

**Comment by windauer** [comment]
Current live version on all servers is: 79a7eb6aca1d847065574e93d09ab9710fc2c986

**Comment by windauer** [comment]
current live version on  https://gf.tei-publisher.com/ is 62dbf604ab639c86f7180bdfa924418f567a7857 (branch `iteration-2`). https://johannconradfischer.sources-online.org/ is untouched.

**Comment by windauer** [comment]
current live version on  https://gf.tei-publisher.com/ is 1dc920b8f8e65bd584f77271b57b447274350c47 from Wed Sep 20 17:00:15 2023 +0200

**Comment by windauer** [comment]
current live version on https://gf.tei-publisher.com/ is #8db68d9dd2bdf2b1668f0f5fc378a8962f583648 from Thu Sep 28 10:22:08 2023 +0200

**Comment by windauer** [comment]
current live version on  https://gf.tei-publisher.com/ is e471b464e897621587cdf7d15649bba0992dd110 from Thu Sep 28 16:25:37 2023 +0200

**Comment by windauer** [comment]
current live version on https://gf.tei-publisher.com/ is c7e53ef1fb0f2d645a14d10744d4b8718d453d28 - server was updated at Mon Oct 9 17:40 CET.

**Comment by windauer** [comment]
current live version on https://gf.tei-publisher.com/ is 0d10df4d422b359af4a9e81cd8ffa1917b1e344f from Tue Oct 17 11:45:46 2023 +0200. The register data in this repo got updated as well

**Comment by windauer** [comment]
current live version on https://gf.tei-publisher.com/ is  f6ba27425f3af5f7d8975c92e0ada4fa5e2dbe46 from Wed Oct 18 22:11:14 2023 +0200. The register data in this repo got updated to version from 2023-10-19 12:05:31 as well.

**Comment by windauer** [comment]
ich werde in kürze ein Update auf https://www.johannconradfischer.com durchführen und dabei den Bugfix #496 auszurollen.

**Comment by windauer** [comment]
deployed fix for #495 to https://gf.tei-publisher.com/en/keywords/gfa-keywords-8875 (only required update of Anton data in gf-publisher app).

**Comment by windauer** [comment]
current live version on https://gf.tei-publisher.com/ is 4add5201eeb2fbfe7c77711203152eb6d25861fa from  Mon Oct 30 15:46:11 2023 +0100. The register data in this repo got updated as well

**Comment by HelenaSabel** [comment]
current live version on https://gf.tei-publisher.com/ is fa04e9b5dcb6bf5b59af91b46d11fd22487b5528 from Fri Nov 3 14:19:20 2023 +0000. The register data in this repo got updated as well

**Comment by HelenaSabel** [comment]
current live version on https://gf.tei-publisher.com/ isd51b882e79bf886d2b81af11bc3e07e63e92b592 from Fri Nov 17 10:54:20 2023 +0000. The register data  (and the i18n localization files) in this repo got updated as well

**Comment by feggimann** [comment]
@windauer @HelenaSabel when updating the last live version on the test server the register data has not been filtered to show only register data used in the edition. At the moment all data from Anton is visible. Please update and show only register data for edition asap! otherwise we cannot check completeness of data. Thank you!

**Comment by HelenaSabel** [comment]
@feggimann done!

**Comment by HelenaSabel** [comment]
current live version on https://gf.tei-publisher.com/ is 473a7210e9bcc156fc6f6e339d61bb05519232a9 from Tue Dec 5 15:47:41 2023 +0100.  The register data in this repo got updated as well.

**Comment by HelenaSabel** [comment]
current live version on https://gf.tei-publisher.com/ is 7cae4b6f85428c6fab64f98f2e765662dac82c97 from Mon Dec 11 15:46:35 2023 +0100.  The register data in this repo got updated as well.

**Comment by HelenaSabel** [comment]
Current live version on http://johannconradfischer.com is 67029d8437f8ad7b2211e73b1211421d8c87e399 from Fri Dec 15 13:00:58 2023 +0100, which includes the register data downloaded on 2023-12-15 (~ 11:42).
